lib/src/view/game/game_body.dart

```diff
@@ -119,7 +119,7 @@ class GameBody extends ConsumerWidget {
     return gameStateAsync.when(
       data: (gameState) {
         final position = gameState.game.positionAt(gameState.stepCursor);
-        final sideToMove = position.turn;
+        final sideToMove = gameState.game.sideToMove;
```

Comment:
The variable name adds confusion. 

I'd rather not declare it and pass `gameState.game.sideToMove` directly where it's needed.

Reply:
Now I have fixed it.